developer-docs-site/docs/nodes/validator-node/using-gcp.md

```diff
@@ -121,10 +121,10 @@ This will download all the Terraform dependencies for you, in the `.terraform` f
 
     ```
     aptos genesis set-validator-configuration \
-        --keys-dir ~/$WORKSPACE 
-        --local-repository-dir ~/$WORKSPACE 
-        --username <pick a username for your node> 
-        --validator-host $VALIDATOR_ADDRESS:6180 
+        --keys-dir ~/$WORKSPACE \
+        --local-repository-dir ~/$WORKSPACE \ 
+        --username <pick a username for your node> \ 
```

Comment:
nit eliminate all white space at end of lines

Reply:
Fixed, thanks.